commands/build.js

```diff
@@ -155,13 +156,20 @@ module.exports = {
         data: JSON.stringify({
           name,
           description,
+          repository,
           xr_type: xrType,
           start_url: startUrl,
           xr_details: xrDetails,
         }, null, 2),
       },
     ];
 
+    // Default to package name if not explicitly provided
+    argv.output = argv.output || `${name}.wbn`;
+
+    // Ensure output file extension is .wbn
+    if (!argv.output.endsWith('.wbn')) argv.output += '.wbn';
```

Comment:
Actually... I don't know why this is here. If the user already expressed a different file name we probably shouldn't be deciding that it's going to be a different extension instead.